pkg/querybuilder/resourcefilter/tables.go

```diff
@@ -0,0 +1,8 @@
+package resourcefilter
+
+const (
+	TracesDBName             = "signoz_traces"
+	TraceResourceV3TableName = "distributed_traces_v3_resource"
+	LogsDBName               = "signoz_logs"
```

Comment:
nit: combine the db names to a single source if possible ?

Reply:
Probably, Where do suggest we put them.